akka-persistence/src/main/scala/akka/persistence/journal/leveldb/LeveldbIdMapping.scala

```diff
@@ -66,7 +66,7 @@ private[persistence] trait LeveldbIdMapping extends Actor { this: LeveldbStore =
   private def writeIdMapping(id: String, numericId: Int): Int = {
     idMap = idMap + (id -> numericId)
     leveldb.put(keyToBytes(mappingKey(numericId)), id.getBytes(UTF_8))
-    newPersistenceIdAdded(id)
+    this.newPersistenceIdAdded(id)
```

Comment:
This change I'm not too sure either.

The compiler error is:

```

[error] -- [E049] Reference Error: /Users/ignasi/git/github/akka/akka/akka-persistence-scala3/akka-persistence/src/main/scala/akka/persistence/journal/leveldb/LeveldbIdMapping.scala:69:4 
[error] 69 |    newPersistenceIdAdded(id)
[error]    |    ^^^^^^^^^^^^^^^^^^^^^
[error]    |    Reference to newPersistenceIdAdded is ambiguous,
[error]    |    it is both defined in trait LeveldbIdMapping
[error]    |    and inherited subsequently in trait LeveldbIdMapping

```

And the cause is:

```
private[persistence] trait LeveldbIdMapping extends Actor { this: LeveldbStore =>
```

where `LeveldbStore` extends `LeveldbIdMapping` overriding `newPersistenceIdAdded`.

All this code is on the LevelDb journal which is deprecated (https://github.com/akka/akka/pull/30213) and it's `private[akka]`. I'm tempted to merge the two traits `LeveldbIdMapping` and `LeveldbStore` into one.